src/sentry/static/sentry/app/views/organizationIntegrations/integrationAlertRules.jsx

```diff
@@ -25,34 +25,33 @@ class IntegrationAlertRules extends React.Component {
 
     return (
       <Panel>
-        <PanelHeader disablePadding>
-          <Box pl={2}>{t('Project Configuration')}</Box>
-        </PanelHeader>
+        <PanelHeader>{t('Project Configuration')}</PanelHeader>
         <PanelBody>
           {projects.length === 0 && (
             <EmptyMessage size="large">
               {t('You have no projects to add Alert Rules to')}
             </EmptyMessage>
           )}
           {projects.map(project => (
-            <PanelItem key={project.slug} alignItems="center">
-              <Box flex="1">
-                <ProjectBadge project={project} avatarSize={16} />
-              </Box>
-              <Box pr={1}>
-                <Button
-                  to={`/settings/${orgId}/projects/${project.slug}/alerts/rules/new/`}
-                  size="xsmall"
-                >
-                  {t('Add Alert Rule')}
-                </Button>
-              </Box>
-            </PanelItem>
+            <ProjectItem key={project.slug}>
+              <ProjectBadge project={project} avatarSize={16} />
+              <Button
+                to={`/settings/${orgId}/projects/${project.slug}/alerts/rules/new/`}
+                size="xsmall"
+              >
+                {t('Add Alert Rule')}
+              </Button>
+            </ProjectItem>
           ))}
         </PanelBody>
       </Panel>
     );
   }
 }
 
+const ProjectItem = styled(PanelItem)`
+  align-items: center;
+  justify-content: space-between;
```

Comment:
Please, and when there is no space (on mobile devices, for example) ... they would be very close to each other ... are you sure about this change?

Reply:
Yeah, I think we should revisit this later as we have this pattern in a few other places as well.